docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>Do you need to set the dtype here?</p><p> </p><p>Why is it .index[-1][0]+2?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>yep, without it, I was getting a warning from numpy. Because linspace function returns&nbsp;n&nbsp;evenly spaced samples, calculated over the interval we provide, and since the 0 and 2 pi are the same points, so we needed to add 1 for that( and 1 was already there from the index to number conversion), so a total of 2.</p><p> </p>